src/Model/ResourceField.php

```diff
@@ -55,7 +56,14 @@ class ResourceField extends DataObject
     public function getCMSFields()
     {
         $this->beforeUpdateCMSFields(function (FieldList $fields) {
-            $fields->removeByName('Name');
+            $originalTitle = ReadonlyField::create('Name', i18n::_t(
+                __CLASS__ . '.ORIGINAL_TITLE',
+                'Original title'
+            ))->setDescription(i18n::_t(
```

Comment:
We tend to use the global function `_t()` over the namespaced version (if we're talking about consistency)

Reply:
Yup.
I noticed that - but only after having done all this 😂 
I'll swap them back & drop the `use` statement at some point in the future :)